recipes/libcurl/all/conanfile.py

```diff
@@ -68,13 +68,13 @@ def imports(self):
 
     def config_options(self):
         if not tools.is_apple_os(self.settings.os):
-            self.options.remove("darwin_ssl")
```

Comment:
is it undocumented as `self.requires.add`? or why changing it?

Reply:
more pythonic, we talked about it few time, also, the docs uses this version: https://docs.conan.io/en/latest/mastering/conditional.html#constrain-settings-and-options